src/styles/tailwind.css

```diff
@@ -0,0 +1,42 @@
+@import "./index/hero.css";
+@import "./index/about.css";
+@import "./index/joinus.css";
+@import "./index/project_card.css";
+@import "./index/our_projects.css";
+@import "./index/our_members.css";
+@import "./layout/header.css";
+@import "./events/events.css";
+@import "./resources/resources.css";
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+@tailwind aspect-ratio;
+
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Ubuntu:wght@400;700&display=swap');
+
+.bg-general{
+    background: url('https://i.imgur.com/tLt6Zo6.png'), #F4A55D;
```

Comment:
Why is this resource external while others are being served locally? I would rather having all resources served in a single fashion.